### DISCUSSION TITLE
Removing # from the generated url of Quasar CLI app

### DISCUSSION THREAD

**Discussion by Ammamon** [quasarframework/quasar#9730] [General - Components / Directives / etc] [answered]
How to remove # from the generated url of Quasar CLI app?

**Comment by rstoenescu** [comment, marked as answer]
Switch to Vue Router "history" mode instead.
quasar.conf.js > build > vueRouterMode: 'history'

Make sure that you read about 'history' mode first and understand the difference between it and 'hash'. Deployment webserver needs to be correctly configured in order for it to be able to work.

**Reply by Ammamon** [reply]
Thank you very much for the help.

**Comment by tattwamasii2p** [comment]
Docs on this: https://router.vuejs.org/guide/essentials/history-mode.html
